src/ol/interaction/selectinteraction.js

```diff
@@ -264,7 +264,7 @@ ol.interaction.Select.handleEvent = function(mapBrowserEvent) {
          * @param {ol.layer.Layer} layer Layer.
          */
         function(feature, layer) {
-          var index = goog.array.indexOf(features.getArray(), feature);
+          var index = features.getArray().indexOf(feature);
           if (index == -1) {
```

Comment:
Now in https://github.com/fredj/ol3/commit/da611b9cb54ca6627b9b07a2a89a2447002a0531